scripts/kind-two-node-cluster.yaml

```diff
@@ -2,11 +2,5 @@ kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
   - role: control-plane
-    kubeadmConfigPatches:
```

Comment:
any reason for removing this?

Reply:
Adam and I were trying to debug an issue with his KIND cluster not being created properly. I noticed this unnecessary config patch for the cluster and we deleted it just to make sure it wasn't affecting him. It's not necessary for the e2e tests, so we figured we'd just leave it out.